packages/cli/src/commands/items/list.ts

```diff
@@ -1,22 +1,60 @@
-import { ItemService } from '@meeco/sdk';
+import { IItemListFilterOptions, ItemService } from '@meeco/sdk';
+import { flags as _flags } from '@oclif/command';
 import { AuthConfig } from '../../configs/auth-config';
 import authFlags from '../../flags/auth-flags';
 import pageFlags from '../../flags/page-flags';
 import MeecoCommand from '../../util/meeco-command';
 
 export default class ItemsList extends MeecoCommand {
   static description = 'List the items that a user has in their vault';
-  static examples = [`meeco items:list -a path/to/auth.yaml`];
+  static examples = [
+    `meeco items:list -a path/to/auth.yaml`,
+    `meeco items:list --templateIds e30a36a5-6cd3-4d58-b838-b3a96384beab -a path/to/auth.yaml`,
+    `meeco items:list --classificationSchemeName esafe -a path/to/auth.yaml`,
+    `meeco items:list --classificationNodeName pets -a path/to/auth.yaml`,
+    `meeco items:list --classificationNodeName --classificationNodeNames pets,vehicles -a path/to/auth.yaml`,
+  ];
 
   static flags = {
     ...MeecoCommand.flags,
     ...authFlags,
     ...pageFlags,
+    templateIds: _flags.string({
+      required: false,
+      description: 'a list of template IDs separated by commas',
+    }),
+    classificationSchemeName: _flags.string({
```

Comment:
I guess that's more a question of "Should the CLI flags follow the API naming scheme or be more ergonomic to type"?

I'm not really fussed either way but these were copied verbatim from the API SDK naming which in turn comes from swagger.

Reply:
IMO the CLI should help in understanding the concepts under the API rather than exposing backend implementation details that already lurk in the swagger...

Reply:
I can get behind that. Let's go with the shorter names then.

Reply:
- yep, make sense. I will change it.